app/components/ImageInput/index.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Avatar from '@material-ui/core/Avatar';
+import TextField from '@material-ui/core/TextField';
+import withStyles from '@material-ui/core/styles/withStyles';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import AttachFileIcon from '@material-ui/icons/AttachFile';
+
+const styles = theme => ({
+  hidden: { display: 'none' },
+  avatar: {
+    cursor: 'pointer',
+    borderRadius: 0,
+    marginBottom: '25%',
+    backgroundColor: theme.palette.primary.main,
+  },
+});
+
+const ImageInput = props => {
+  const {
+    classes,
+    name,
+    label,
+    type,
+    setFieldValue,
+    showFileUpload,
+    handleImageChange,
+    fileUpload,
+  } = props;
+
+  const renderTextField = () => (
+    <TextField
+      InputProps={{
+        endAdornment: (
+          <InputAdornment position="end">
+            <Avatar className={classes.avatar} onClick={showFileUpload}>
+              <AttachFileIcon />
+            </Avatar>
+          </InputAdornment>
+        ),
+      }}
+      name={name}
+      type={type}
+      label={label}
+      setFieldValue={setFieldValue}
+      fullWidth
+    />
+  );
+
+  return (
+    <div>
+      {renderTextField()}
+      <input
```

Comment:
Why do you have both a text input and a file input? You need only the file input.